tasks/cockpit-tasks-webhook.yaml

```diff
@@ -57,3 +73,79 @@ items:
       name: webhook
     port:
       targetPort: 8080
+
+- kind: Service
+  apiVersion: v1
+  metadata:
+    name: amqp
+  spec:
+    clusterIP: None
+    selector:
+      infra: webhook
+    ports:
+    - targetPort: 5671
+      port: 5671
+      protocol: TCP
+      name: amqp-secure
+
+- kind: Route
+  apiVersion: v1
+  metadata:
+    name: amqp
+  spec:
+    to:
+      kind: Service
+      name: amqp
+    port:
+      targetPort: amqp-secure
+    tls:
+      termination: passthrough
+
+- kind: ConfigMap
+  apiVersion: v1
+  metadata:
+    name: amqp-config
+  data:
+    definitions.json: |
```

Comment:
YAML does have comments, so this would be a good place to mention that the password cannot actually be used, thus exposing it doesn't matter.